images/runtime/php-fpm/template.base.Dockerfile

```diff
@@ -93,7 +93,7 @@ RUN set -eux; \
 #  - https://docs.microsoft.com/en-us/sql/connect/php/installation-tutorial-linux-mac
```

Comment:
When looking above in the source file, line 80, I can still see the condition:
`if [[ $PHP_VERSION != 8.* ]]; then \`
Is this only going to work for PHP 7.4 in that case?

Reply:
I verified that the php imagick sample apps passed for 7.4, 8.0 and 8.1 with these installations. So here I kept the original way to install imagick library for 8.0 and 8.1.

Reply:
Sorry but I'm a bit confused: could you please clarify where in the code we are installing imagick for PHP 8.0 and 8.1? The only place where I see a reference to imagick in this file is line 81, that explicitely excludes PHP 8.*.
Could you please clarify what I'm missing?

Reply:
Good catch, this is very strange. Here's the place where we install the imagick library but I wonder why the tests all passing even with this condition. So I'll remove the condition here and check if the tests are correctly covered.

Reply:
Thanks!